Goal: Adjustments in freebusy template

Author: Peter Eisentraut <petere@debian.org>

Upstream status: Specific to Debian

Index: b/templates/freebusy.conf.template.in
===================================================================
--- a/templates/freebusy.conf.template.in
+++ b/templates/freebusy.conf.template.in
@@ -1,5 +1,5 @@
 KOLAB_META_START
-TARGET=@freebusy_confdir@/config.php
+TARGET=/etc/kolab/freebusy/config.php
 PERMISSIONS=0640
 OWNERSHIP=@freebusy_usr@:@freebusy_grp@
 KOLAB_META_END
@@ -44,8 +44,8 @@
 /* Horde::Log configuration */
 $conf['log']['enabled']          = true;
 $conf['log']['priority']         = PEAR_LOG_DEBUG; // Leave this on DEBUG for now. We just restructured the package...
-$conf['log']['type']             = 'file';
-$conf['log']['name']             = '@freebusy_logdir@/freebusy.log';
+$conf['log']['type']             = 'syslog';
+$conf['log']['name']             = 'cons, pid';
 $conf['log']['ident']            = 'Kolab Free/Busy';
 $conf['log']['params']['append'] = true;
 
